packages/core/src/components/spinner/_spinner.scss

```diff
@@ -106,8 +106,8 @@ Styleguide components.progress.spinner.js.svg
 
 $spinner-track-color: rgba($gray1, 0.2) !default;
 $spinner-head-color: rgba($gray1, 0.8) !default;
-$dark-spinner-track-color: rgba($gray5, 0.2) !default;
-$dark-spinner-head-color: $gray1 !default;
+$dark-spinner-track-color: rgba($black, 0.3) !default;
```

Comment:
track contrast with the background is very low. can we make this stand out a little more @pkwi?